packages/core/src/stylable-transformer.ts

```diff
@@ -98,18 +96,7 @@ export const transformerWarnings = {
     UNKNOWN_PSEUDO_ELEMENT(name: string) {
         return `unknown pseudo element "${name}"`;
     },
-    IMPORT_ISNT_EXTENDABLE() {
-        return 'import is not extendable';
-    },
-    CANNOT_EXTEND_UNKNOWN_SYMBOL(name: string) {
-        return `cannot extend unknown symbol "${name}"`;
-    },
-    CANNOT_EXTEND_JS() {
-        return 'JS import is not extendable';
-    },
-    UNKNOWN_IMPORT_ALIAS(name: string) {
-        return `cannot use alias for unknown import "${name}"`;
-    },
+    ...STPart.diagnostics,
```

Comment:
In the processor you used the feature's diagnostics object and here you spread it, it should be the same all around

Reply:
fixed